certbot-apache/certbot_apache/tls_configs/current-options-ssl-apache.conf

```diff
@@ -0,0 +1,27 @@
+# This file contains important security parameters. If you modify this file
+# manually, Certbot will be unable to automatically provide future security
+# updates. Instead, Certbot will print and log an error message with a path to
+# the up-to-date file that you will need to refer to when manually updating
+# this file.
+
+SSLEngine on
+
+# Intermediate configuration, tweak to your needs
+SSLProtocol             all -SSLv2 -SSLv3
+SSLCipherSuite          ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS
+SSLHonorCipherOrder     on
+SSLCompression          off
+SSLSessionTickets       off
+
+SSLOptions +StrictRequire
+
+# Add vhost name to log entries:
+LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\"" vhost_combined
+LogFormat "%v %h %l %u %t \"%r\" %>s %b" vhost_common
+
+#CustomLog /var/log/apache2/access.log vhost_combined
+#LogLevel warn
+#ErrorLog /var/log/apache2/error.log
+
+# Always ensure Cookies have "Secure" set (JAH 2012/1)
+#Header edit Set-Cookie (?i)^(.*)(;\s*secure)??((\s*;)?(.*)) "$1; Secure$3$4"
```

Comment:
I can't remember the historical reason of existence of the commented-out configuration directives, but I would think that this would be a good place to get rid of them. 

I wonder if @bmw has a clearer view on this.

Reply:
Waiting for @bmw comment here.

Reply:
I have no idea why this is here. Looks like it predates all of our involvement in the project besides Seth by at least a few years as it was added in https://github.com/certbot/certbot/commit/68f85d9f1abc2b66a6a9a0bd970d42f37395f217.

JAH in the referenced comment is https://jhalderm.com/ who was James' Ph.D advisor. Maybe setting "Secure" on cookies is a good idea (although it seems like it'd break sites that still use HTTP?), but this line has been commented and doing nothing but cluttering up our config for 7 years so I think it can be deleted for now.

I'll ask if others think we should be doing something like this in the future separately.

Reply:
Doing this is tracked in https://github.com/certbot/certbot/issues/7253. This definitely doesn't seem like something we'll ever enable by default by putting it in this static file.

Reply:
OK, I removed the commented lines from the config files.

Reply:
While we're at it, should we unify the configs by removing it from all of the files and adding the new hashes to list within this PR?